integrations/gitlab/.port/spec.yaml

```diff
@@ -29,3 +29,8 @@ configurations:
     type: boolean
     description: If set to true, will use system hook instead of project hooks.
     default: false
+  - name: tokenGroupHooksOverrideMapping
+    required: false
+    type: object
+    description: "Mapping of Gitlab tokens to groups in which to create webhooks, if not set, it will create webhooks only on root groups. Example: {\"THE_GROUP_ADMIN_TOKEN\":[\"GROUP1_FULL_PATH\", \"GROUP2_FULL_PATH\"]}"
```

Comment:
Root groups is not correct
Its will create the hooks for the highest possible group

Reply:
Doesnt that mean root?